lib/ClimaLandSimulations/src/Fluxnet/fluxnet_sites/US-Ha1/US-Ha1_parameters.jl

```diff
@@ -0,0 +1,126 @@
+export harvard_default_params
+
+"""
+Site-specific model parameters for running CliMA LSM on the Harvard
+fluxtower site.
+"""
+harvard_default_params = (
+    hetero_resp = (
+        ## LAI data from MODIS
+        # Heterotrophic respiration parameters
+        θ_a100 = FT(0.1816),
+        D_ref = FT(1.39e-5),
+        b = FT(4.547),
+        D_liq = FT(3.17),
+        # DAMM
+        α_sx = FT(194e3),
+        Ea_sx = FT(61e3),
+        kM_sx = FT(5e-3),
+        kM_o2 = FT(0.004),
+        O2_a = FT(0.209),
+        D_oa = FT(1.67),
+        p_sx = FT(0.024),
+    ),
+    auto_resp = (
+        # Autotrophic respiration parameters
+        ne = FT(8 * 1e-4),
+        ηsl = FT(0.01),
+        σl = FT(0.05),
+        μr = FT(1.0),
+        μs = FT(0.1),
+        f1 = FT(0.012),
+        f2 = FT(0.25),
+    ),
+    soil = (
+        # Soil parameters
+        soil_ν = FT(0.5), # m3/m3
+        soil_K_sat = FT(4e-7), # m/s, matches Natan
+        soil_S_s = FT(1e-3), # 1/m, guess
+        soil_vg_n = FT(2.05), # unitless
+        soil_vg_α = FT(0.04), # inverse meters
+        θ_r = FT(0.067), # m3/m3, from Wang et al. 2021 https://doi.org/10.5194/gmd-14-6741-2021
+    ),
+    soil_heat = (
+        # Soil heat transfer parameters; not needed for hydrology only test
+        ν_ss_quartz = FT(0.1),
+        ν_ss_om = FT(0.1),
+        ν_ss_gravel = FT(0.0),
+        κ_quartz = FT(7.7), # W/m/K
```

Comment:
for US-Ha, these params appear here, but for Ozark, they appear in a different struct. could make consistent [preferably this PR]

Reply:
sorry, only ozark was up to date. should be good now.